backend/wine_map/models.py

```diff
@@ -0,0 +1,96 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.core.validators import MaxValueValidator, MinValueValidator
+
+
+class User(AbstractUser):
+    favourite_wines = models.ManyToManyField("Wine")
+    groups = models.ManyToManyField("auth.Group", related_name="wine_map_users")
+    user_permissions = models.ManyToManyField(
+        "auth.Permission",
+        related_name="wine_map_users",
+        blank=True,
+    )
+
+    def __str__(self) -> str:
+        return self.username
+
+
+class Wine(models.Model):
+    WINE_TYPES = [
+        ("red", "Red"),
+        ("white", "White"),
+        ("rose", "Rosé"),
+    ]
+
+    name = models.CharField(max_length=255)
+    brand = models.ForeignKey("Brand", on_delete=models.CASCADE, related_name="brand")
+    wine_type = models.CharField(choices=WINE_TYPES, max_length=5)
+    country = models.ForeignKey("Country", on_delete=models.CASCADE)
+    sweetness = models.ManyToManyField("Sweetness")
+    taste = models.ManyToManyField("Taste")
```

Comment:
If a wine can have multiple tastes:
```suggestion
    tastes = models.ManyToManyField("Taste")
```
If only one:
```suggestion
    taste = models.ForeignKey("Taste")
```